pkg/services/ngalert/api/tooling/definitions/provisioning_mute_timings.go

```diff
@@ -79,3 +95,30 @@ func (mt *MuteTimeInterval) ResourceType() string {
 func (mt *MuteTimeInterval) ResourceID() string {
 	return mt.MuteTimeInterval.Name
 }
+
+type MuteTimeIntervalExport struct {
+	OrgID                   int64
```

Comment:
Nit: Should this have JSON tag `json:"orgId"` so it renders in a consistent casing?

Reply:
yep! Thanks for pointing out!